dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -27,32 +27,39 @@ class SentryStackTraceFactory {
             ? Chain.parse(stackTrace)
             : Chain.parse('');
 
-    // TODO: if strip symbols are enabled, thats what we see:
-    // warning:  This VM has been configured to produce stack traces that violate the Dart standard.
-    // ***       *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
-    // unparsed  pid: 30930, tid: 30990, name 1.ui
-    // unparsed  build_id: '5346e01103ffeed44e97094ff7bfcc19'
-    // unparsed  isolate_dso_base: 723d447000, vm_dso_base: 723d447000
-    // unparsed  isolate_instructions: 723d452000, vm_instructions: 723d449000
-    // unparsed      #00 abs 000000723d6346d7 virt 00000000001ed6d7 _kDartIsolateSnapshotInstructions+0x1e26d7
-    // unparsed      #01 abs 000000723d637527 virt 00000000001f0527 _kDartIsolateSnapshotInstructions+0x1e5527
-    // unparsed      #02 abs 000000723d4a41a7 virt 000000000005d1a7 _kDartIsolateSnapshotInstructions+0x521a7
-    // unparsed      #03 abs 000000723d624663 virt 00000000001dd663 _kDartIsolateSnapshotInstructions+0x1d2663
-    // unparsed      #04 abs 000000723d4b8c3b virt 0000000000071c3b _kDartIsolateSnapshotInstructions+0x66c3b
-    // unparsed      #05 abs 000000723d5ffe27 virt 00000000001b8e27 _kDartIsolateSna...
-
-    // abs => instruction_addr
-    // build_id => debug_images
-
     final frames = <SentryStackFrame>[];
+    var symbolicated = true;
+
     for (var t = 0; t < chain.traces.length; t += 1) {
-      final encodedFrames = chain.traces[t].frames
-          // we don't want to add our own frames
-          .where((frame) => frame.package != 'sentry')
-          .map((f) => encodeStackTraceFrame(f));
+      final trace = chain.traces[t];
 
-      frames.addAll(encodedFrames);
+      for (final frame in trace.frames) {
+        // we don't want to add our own frames
+        if (frame.package == 'sentry') {
```

Comment:
Should this be part of the default InAppExclude list instead?

Reply:
on Android and Cocoa, we don't even add them